packages/client/src/forms/register/mappings/event-specific-fields/birth/query/registration-mappings.ts

```diff
@@ -33,6 +37,23 @@ export function getBirthRegistrationSectionTransformer(
   if (queryData[sectionId].status) {
     transformStatusData(transformedData, queryData[sectionId].status, sectionId)
   }
+
+  if (
+    Array.isArray(queryData[sectionId].certificates) &&
+    queryData[sectionId].certificates.length > 0
+  ) {
+    const certificate = queryData[sectionId].certificates.slice(-1)[0]
+    // since we shall need this certificate only for ready to issue tab, to calculate certificate fee
+    transformedData[sectionId].certificates = certificate?.certificateTemplateId
+      ? [
+          {
+            templateConfig: offlineData?.templates.certificates?.find(
+              (x) => x.id === certificate.certificateTemplateId
+            )
+          }
+        ]
+      : []
+  }
```

Comment:
Could you keep this only as an id reference all to way to where the certificate is printed? Or all to way to the UI component where you need to stow details of the certificate?

Reply:
As agreed in previous conversation, will remove the `templateConfig` and just keep the id.

Reply:
Yes, as agreed in the above conversation, will do.